### PR TITLE
fix: Added onClick event listener to show Appearance Menu options in mobile

### PR DESCRIPTION
**Pull request by jvmsangkal**
Changes:
=====
- Added onClick event listener to show Appearance Menu options in mobile

### CODE REVIEW

File: app/components/ContextMenu/index.js [hunk 1]
```diff
@@ -99,8 +99,7 @@ const Background = styled.div`
   border-radius: 6px;
   padding: 6px 0;
   min-width: 180px;
-  overflow: hidden;
-  overflow-y: auto;
```
Comment (tommoor) [inline app/components/ContextMenu/index.js:103]: I'm not sure when this regressed yet, but this won't be the fix – the appearance menu should be a portal so the overflow of the parent has no effect. We still need these rules to ensure that the content of the menu scrolls correctly when it won't fit.

Comment (tommoor) [reply app/components/ContextMenu/index.js:103]: I bisected and amazingly it was this change: https://github.com/outline/outline/commit/8009e8f6917ae03a8a00a7d8ec0a02ca42bbc739

Fixed on main as this is a pretty serious issue:
https://github.com/outline/outline/commit/974d6b2cbed8476d771f887211bea93ba6a6394e

I think we could keep this PR with your improvement for the click behavior though…

Comment (jvmsangkal) [reply app/components/ContextMenu/index.js]: Thank you! I've updated the PR title and description and removed the commit relating to this fix.

**Comment by tommoor** [comment]
Thanks 😄

**Pull request merged**